Server/controllres/search.js

```diff
@@ -0,0 +1,11 @@
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
```

Comment:
you've already doing this in another file, instead of repeating your self, you can turn this into a helper module or something like 

fetch.js
```js
const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args)); // not sure what is this for yet

module.exports = fetch
```
then you can require it wherever you need it